sources/packages/backend/apps/api/src/route-controllers/designation-agreement/models/designation-agreement.dto.ts

```diff
@@ -63,7 +66,13 @@ export class UpdateDesignationAPIInDTO {
 export class SubmitDesignationAgreementAPIInDTO {
   @Allow()
   dynamicData: unknown;
-  @Allow()
+  /**
```

Comment:
I would recommend changing as below. I like the idea of using the form "`locations`" but I believe that would be more for the inline object types, for instance, when we create inline `options` types.
```ts
/**
   * Locations being designated.
   * Must be validated because it is not part of the dynamic data.
   */
```